src/Sulu/Bundle/SecurityBundle/EventListener/LogoutEventSubscriber.php

```diff
@@ -36,14 +36,19 @@ public function __construct(UrlGeneratorInterface $urlGenerator)
 
     public function onLogout(LogoutEvent $logoutEvent): void
     {
-        if (null !== $logoutEvent->getResponse()) {
```

Comment:
Is always set. So to avoid conflicts with community bundle or similar we need to check if we are in the admin context. To avoid context based service this is done via a the pathInfo of the request.